scripts/crdcopy.py

```diff
@@ -0,0 +1,15 @@
+s = "versions:\n"
```

Comment:
perhaps put in a `\n     versions:\n` to ensure if finds the one with the right indentation as well?